hydromt/models/model_lumped.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import sys, os
+from .model_api import Model
+import xarray as xr
+import numpy as np
+import geopandas as gpd
+from shapely.geometry import box
+
+from typing import List
+
+import logging
+import os
+
+__all__ = ["LumpedModel"]
+logger = logging.getLogger(__name__)
+
+
+class LumpedModel(Model):
+    def __init__(
+        self,
+        root: str = None,
+        mode: str = "w",
+        config_fn: str = None,
+        data_libs: List[str] = None,
+        logger=logger,
+    ):
+        # Initialize with the Model class
+        super().__init__(
+            root=root,
+            mode=mode,
+            config_fn=config_fn,
+            data_libs=data_libs,
+            logger=logger,
+        )
+
+    def read(self):
+        """Method to read the complete model schematization and configuration from file."""
+        super().read()
+        # Other specifics to LumpedModel...
+
+    def write(self):
+        """Method to write the complete model schematization and configuration to file."""
+        super().write()
+    
+    def set_forcing_name():
+
+    def space_interpolation():
```

Comment:
def setup_fromraster():
#read raster data
#zonal stats
#add to hydromt object

Reply:
def setup_temporaldata(): #temporal raster and temporal vector
#Check soemthing for precip?
#In the lumped model always do the zonalstats

Reply:
in setup_temporaldata: check resmapling in time and space to check model dimensions